README.md

```diff
@@ -14,4 +14,6 @@ If you want to see others stats as well, open an issue, submit a pull request, o
 
 Run from the command line like `python3 gradstats.py` and all of your dreams will come true.
 
+You may need to run `pip3 install requests` before `python3 gradstats.py`.
```

Comment:
I'd remove the blank line on 16 and reword this to something like "Note that the program relies on the `requests` package, which can be installed using `pip3 install requests`."